src/styles/main.scss

```diff
@@ -0,0 +1,284 @@
+$c-seconds: rgb(0, 255, 45);
+$c-sec-axis: rgb(0, 255, 45);
+$c-sec-glow: rgb(0, 255, 45);
+
+$c-minutes: hsl(131, 75%, 50%);
+$c-min-axis: hsl(131, 75%, 50%);
+$c-min-glow: hsl(131, 75%, 50%);
+
+$c-hours: hsl(131, 50%, 45%);
+$c-hour-glow: hsl(131, 50%, 45%);
+$c-hour-axis: hsl(131, 50%, 45%);
+
+$c-lines: hsla(130, 50%, 50%, 0.68);
+$c-circles: rgba(68, 196, 68, 0.32);
+
+$size-glow: 0.5;
+$size-glow-blur: 0.1;
+
+$size-main: 75;
+$size-circle: 9;
+$size-pad: 70;
+$size-second: 62;
+$size-minute: 63.5;
```

Comment:
Whi not to sent units here?

Reply:
fixed
(I've brought my mind back to units `vmin` only after specific lesson wich went after I've been making this...)))